krux/cli.py

```diff
@@ -139,35 +140,53 @@ def __init__(self, name, parser=None, logger=None, lockfile=False):
         if lockfile:
             self.acquire_lock(lockfile)
 
+    def init_logging(self):
```

Comment:
this is only meant to be called from within this module, so let's mark it as private by prefixing it with ___, like we did for ___release_lock_file